packages/file-settings/README.md

```diff
@@ -1,17 +1,47 @@
-# File Settings
+# `@frontity/file-settings`
 
-### _This is an internal package used by @frontity/core._
+[![Version](https://img.shields.io/npm/v/@frontity/file-settings.svg)](https://www.npmjs.com/package/@frontity/file-settings) [![npm](https://img.shields.io/npm/dw/@frontity/file-settings)](https://www.npmjs.com/package/@frontity/file-settings) [![License: Apache--2.0](https://img.shields.io/badge/license-Apache%202-lightgrey)](https://github.com/frontity/frontity/blob/master/LICENSE)
 
-## Usage
+> This is an internal package used by [@frontity/core](https://github.com/frontity/frontity/tree/dev/packages/core)
 
-This package is used to import the Frontity settings from a local file.
+A settings package for Frontity that reads a `frontity.settings.js` file.
+This package allows to decouple from where the settings are retrieved.
 
-You can install it with `npm`:
+_`frontity.settings.js` should contain data that allows it to be serializable (as a JSON) so it can be stored in DB_
 
-```
+## Table of contents
+
+<!-- toc -->
+
+- [Install](#install)
+- [Usage](#usage)
+- [API Reference](#api-reference)
+  * [`async getSettings(options) => settings`](#async-getsettingsoptions--settings)
+    + [Parameters](#parameters)
+    + [Return](#return)
+  * [`async getAllSites() => sites`](#async-getallsites--sites)
+    + [Return](#return-1)
+- [Settings File](#settings-file)
+- [Typescript](#typescript)
+  * [`Settings`](#settings)
+- [Feature Discussions](#feature-discussions)
+- [Changelog](#changelog)
+- [» Frontity Resources 🌎](#%C2%BB-frontity-resources-%F0%9F%8C%8E)
+- [» Get involved 🤗](#%C2%BB-get-involved-%F0%9F%A4%97)
+
+<!-- tocstop -->
+
+## Install
+
+```sh
 npm i @frontity/file-settings
 ```
 
+
+## Usage
```

Comment:
We should remove this section. I guess Eduardo wrote it back when we started because this was the very first package to be completed and we didn't know how or where to document things. But all this is not relevant anymore, and I'd say it's even confusing as it seems like people should do these things and use these APIs, when in reality it's the Frontity framework the one using them internally.